gn/BUILD.gn

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2019 Google, Inc.
+#
+# All rights reserved.
```

Comment:
There is a license mismatch here. This project uses Apache 2 as per Khronos CLA.

Reply:
done